test/cider/piggieback_test.clj

```diff
@@ -10,13 +10,6 @@
 (def ^:dynamic *server-port* nil)
 (def ^:dynamic *session*)
 
-(defn assert-exit-ns [session ns]
-  (let [v (keep :ns (nrepl/message
-                     session
-                     {:op "eval" :code "clojure.core/*ns*"}))]
-    (assert (= ["user"] v)
-            (pr-str v))))
```

Comment:
I've spent a lot of time trying to make this work for 1.3, and it turned it it was failing for the wrong reason. With nREPL 1.3, sending an additional `eval` command to the same session after `:cljs/quit` is sent fails with an IOException because apparently some input or output stream gets closed after quitting. I can't judge whether this is good or bad as I don't understand the desired interactions between CLJ and CLJS REPL. If somebody comes along and reports the exact issue, I'll be happy to fix it, but right now I've no idea if the session should survive after `:cljs/quit`.

Reply:
I recall that the session was supposed to become an ordinary Clojure session after `:cljs/quit`. (as ClojureScript sessions start their lives as ordinary Clojure sessions)